CONTRIBUTING.md

```diff
@@ -14,9 +14,9 @@ From the [GitHub Help Page of Using Pull Requests](https://help.github.com/artic
 
 The cBioPortal currently maintains three branches:
 
- * **master**:  this is the most stable branch, reflecting that currently running in production on cbioportal.org.
- * **rc**:  release candidate branch, reflecting the next release candidate we are readying for deployment.  Once deployed to production, the rc branch is merged into master.
- * **hotfix**:  hot fix branch, used only for mission critical bug fixes that must be pushed out immediately.  Once deployed to production, the hotfix branch is merged into master. 
+ * **master**:  this reflects what is currently running in production on cbioportal.org. Bug fixes and documentation fixes go here.
+ * **rc**:  release candidate branch, incorporating all the latest features. You could see our **rc** branch as a development branch where we only accept high quality contributions. Once ready for testing on cbioportal.org/beta a new branch is formed with the name **release-x.y.z**.
+ *  **release-x.y.z**: before each release a new branch is created from **rc** that has a name like **release-x.y.z** .This one is usually deployed to www.cbioportal.org/beta. 
```

Comment:
I'm not sure whether this suggests anything about the version numbers of bugfix releases and the semantics of the three numbers, but clarifying that probably wasn't the purpose of this paragraph anyway.

Reply:
yes we should prolly have some explanation of that as well. We can address that in a separate PR